### PR TITLE
removing CircleCI workflow(not in use)

### PR DESCRIPTION
**Pull request by dautovri**
CircleCI no longer in use

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # EVE is Edge Virtualization Engine
 
-[![CircleCI](https://circleci.com/gh/lf-edge/eve.svg?style=svg)](https://circleci.com/gh/lf-edge/eve/tree/master)
```
Comment (rvs) [inline README.md:3]: Instead of just deleting this -- can you please provide an equivalent URL for a GitHub actions build process?

**Review by rvs — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dautovri) [reply README.md:3]: We will add a new one from Github Actions after fixing the Build PR workflow.

**Comment by rvs** [comment]
Not sure I understand -- why can't we add it (actually not even add -- replace) in this PR?

File: README.md (hunk 1 above)
Comment (dautovri) [reply README.md:3]: Added GH action one. Publish workflow indicator

**Review by rvs — APPROVED** [review]
LGTM

**Pull request merged**